editor/src/components/canvas/__snapshots__/dom-walker.spec.browser.tsx.snap

```diff
@@ -262,6 +263,7 @@ Object {
       "y": 0,
     },
     "props": Object {
+      "data-paths": "utopia-storyboard-uid/scene-aaa:aaa",
```

Comment:
Hmmm a lot of these seem wrong. Where they should have two paths (according to the `data-uid` prop below) they only have one.

Reply:
I pushed a fix!